docs/SUMMARY.md

```diff
@@ -62,6 +62,7 @@
 
 ## Explanations
 
+* [Simplified flow](dev/compilation/README.md)
```

Comment:
that's the name I would suggest for this entry we had removed. Do you agree with this name, @BourgerieQuentin @yuxizama @aquint-zama ?

once it's agreed, then maybe can:
- move the file to explanations/[the_right_name].md
- and manage the re-links
in his symlink PR

Reply:
Hum not sure Simplified flow is enough descriptive as it not mention compilation?

Reply:
But it's already the Explanations part of the Concrete doc, ie "compilation" is a bit obvious. @yuxizama , what do you think?

Reply:
How about "Compiler workflow"?

Reply:
Would be fine for me. @BourgerieQuentin, and for you ?

Reply:
ok for me

Reply:
done